usage/camp/src/test/java/org/apache/brooklyn/camp/brooklyn/catalog/CatalogYamlEntityTest.java

```diff
@@ -18,15 +18,14 @@
  */
 package org.apache.brooklyn.camp.brooklyn.catalog;
 
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
-import static org.testng.Assert.fail;
+import static org.testng.Assert.*;
```

Comment:
What IDE do you use? Here's IntelliJ: http://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports